content/en/flux/monitoring/events.md

```diff
@@ -52,8 +60,97 @@ Sample of a `Normal` event produced by kustomize-controller:
+
+Example of a `Warning` event produced by source-controller:
+
+```json
+{
+    "apiVersion": "v1",
+    "count": 4,
+    "eventTime": null,
+    "firstTimestamp": "2023-08-22T20:24:06Z",
+    "involvedObject": {
+        "apiVersion": "source.toolkit.fluxcd.io/v1",
+        "kind": "GitRepository",
+        "name": "podinfo",
+        "namespace": "default",
+        "resourceVersion": "1284973",
+        "uid": "2c2ed1da-556f-4793-863d-7d96e8bab3f5"
+    },
+    "kind": "Event",
+    "lastTimestamp": "2023-08-22T20:24:18Z",
+    "message": "failed to checkout and determine revision: unable to clone 'https://github.com/stefanprodan/podinfo': couldn't find remote ref \"refs/tags/v1.8.9\"",
+    "metadata": {
+        "creationTimestamp": "2023-08-22T20:24:06Z",
+        "name": "podinfo.177dce48bc7db3a4",
+        "namespace": "default",
+        "resourceVersion": "1285016",
+        "uid": "3c8f568a-c99b-4279-8093-6ef08fae325b"
+    },
+    "reason": "GitOperationFailed",
+    "reportingComponent": "",
+    "reportingInstance": "",
+    "source": {
+        "component": "source-controller"
+    },
+    "type": "Warning"
+}
+```
+
+In the above example:
+- The event is about a `GitRepository` named `podinfo` in the `default`
+  namespace, indicated by the `involvedObject` field.
+- The event originates from `source-controller`, indicated by the
+  `source.component` field.
+- The event is a `Warning` type event about a failed Git operation, indicated by
+  the `reason` and `message` fields.
+
 ## Events inspection with kubectl
 
-```shell
-kubectl events -n monitoring --for helmreleaase/kube-prom-stack
+The events associated with a Flux resource can be queried using `kubectl events`
+command:
+
+```console
+$ kubectl events -n flux-system --for kustomization/flux-system
+LAST SEEN   TYPE     REASON                    OBJECT                      MESSAGE
+58m         Normal   ReconciliationSucceeded   Kustomization/flux-system   Reconciliation finished in 448.00332ms, next run in 10m0s
+48m         Normal   ReconciliationSucceeded   Kustomization/flux-system   Reconciliation finished in 486.826649ms, next run in 10m0s
+38m         Normal   ReconciliationSucceeded   Kustomization/flux-system   Reconciliation finished in 502.282127ms, next run in 10m0s
+28m         Normal   ReconciliationSucceeded   Kustomization/flux-system   Reconciliation finished in 543.745587ms, next run in 10m0s
+18m         Normal   ReconciliationSucceeded   Kustomization/flux-system   Reconciliation finished in 465.177441ms, next run in 10m0s
+8m27s       Normal   ReconciliationSucceeded   Kustomization/flux-system   Reconciliation finished in 494.543068ms, next run in 10m0s
 ```
+
+This shows all the events associated with the queried resource in an hour.
+
+## Events inspection with flux CLI
+
+The events associated with a Flux resource can be queried using the `flux
+events` CLI command:
+
+```console
+$ flux events --for Kustomization/flux-system
```

Comment:
Let's add here an example for `flux events --all-namespaces --watch` this command shows you all events issued by Flux, unlike kubectl which can't filter events for an API group.